src/serum-tokenomics.yaml

```diff
@@ -1,19 +1,349 @@
+            key: A
+          - content: MegaSerum (MSRM) is a stack of 1,000,000 SRM
+            key: B
+          - content: MegaSerum (MSRM) is a stack of 100,000 SRM
+            key: C
+          - content: MegaSerum (MSRM) is a stack of 10,000 SRM
+            key: D
+
+      - uuid: 06376e56-5624-4a82-9e27-5ceebaee0b6f
+        content: Select the correct statment(s).
+        answerKeys:
+          - A
+          - D
+        type: MultipleChoice
+        choices:
+          - content: SRM and MSRM are available on both the Solana and Ethereum blockchains
+            key: A
+          - content: All seed sales have short-term lockups
+            key: B
+          - content: You can create 1 SRM by locking up 1 million MSRM
+            key: C
+          - content: 91.25% of all tokens have long-term unlocking periods
+            key: D
+
+  - uuid: 1a06c3b5-35d0-4c8e-9d64-c9b56666ed9a
+    name: Tokenomics (Token Economics)
+    content: |
+      Total Supply: 10,000,000,000 SRM
+      Unlocked Supply: 1,092,844,982 SRM
+      Circulating Supply: 372,158,620 SRM
+      A breakdown of the SRM token supply:
+      - Project Contributors: 22%
+      - Partner and Collaborator Fund: 27%
+      - Team and Advisors: 20%
+      - Locked Seed and Auction Purchaser: 4%
+      - Ecosystem Incentive Fund: 27%
+
+      ## Lockups
+      10% of all SRM tokens begin unlocked. The other 90% all have the same unlock schedule. They are fully locked for the first year, and then unlock linearly over the next 6 years, beginning on the 11th of August 2021, at approximately 1/2190 per day. All seed sales were locked.
+
+      ## Unlocked Token Distribution
+      - The initial circulating supply was 175m tokens, primarily for providing liquidity
+      - Holders of unlocked tokens voluntarily moved tokens into reserves, reducing circulating supply, with plans to re-introduce tokens into circulating only to provide liquidity to extraordinary buy-side demand
+      - 825m was set aside for ecosystem incentives for SRM holders and Serum users
+      - Currently, the circulating supply is approximately 125m
+
+
+    stepItems: []
+  - uuid: 2b0067ea-83a5-49cf-a00c-adda5f33957b
+    name: Evaluation
+    content: |
+    stepItems:
+      - uuid: 188364d9-da34-494c-bb4f-8c1100ecea3e
+        content: What is the total supply of Serum (SRM)?
+        answerKeys:
+          - B
+        type: SingleChoice
+        choices:
+          - content: 1,000,000,000
+            key: A
+          - content: 10,000,000,000
+            key: B
+          - content: 100,000,000,000
+            key: C
+          - content: 100,000,000
+            key: D
+
+      - uuid: 938d0ad8-eeaf-41b5-b022-7faaf51e2982
+        content: What was the initial circulating supply of Serum (SRM)?
+        answerKeys:
+          - C
+        type: SingleChoice
+        choices:
+          - content: 125m tokens
+            key: A
+          - content: 825m tokens
```

Comment:
Please use consistent units in both question. Like `100,000,000,000` or `100 Billion`